tox.ini

```diff
@@ -48,10 +48,13 @@ commands =
     py.test --create-db -n 2 -m 'not es_tests' -v src/olympia/ --ignore src/olympia/addons/ --ignore src/olympia/devhub/ --ignore src/olympia/editors/ --ignore src/olympia/amo/ --ignore src/olympia/users/ {posargs}
 
 [testenv:ui-tests]
-passenv = DISPLAY
-install_command = pip install {packages}
-deps = -rtests/ui/requirements.txt
-commands = pytest --verify-base-url --driver Firefox tests/ui {posargs}
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL ELASTICSEARCH_LOCATION
+install_command = pip install --exists-action=w --no-deps {packages}
+deps =
+    -rtests/ui/requirements.txt
+    -rrequirements/dev.txt
+commands =
+    pytest -n auto --driver Firefox -v tests/ui {posargs}
```

Comment:
https://github.com/mozilla/addons-server/blob/master/Makefile#L129 is what's eventually called